.github/workflows/build.yml

```diff
@@ -148,6 +148,11 @@ jobs:
           cache-dependency-path: "**/package-lock.json"
           node-version: "16"
 
+      - name: Set up dotnet
```

Comment:
⚠️ This is done before checkout in the other usages.

Reply:
Fixed.